pkg/k8s/go.mod

```diff
@@ -69,8 +69,8 @@ require (
 
 // has to be in sync with cilium/cilium
 replace (
-	github.com/miekg/dns => github.com/cilium/dns v1.1.51-0.20220729113855-5b94b11b46fc
+	github.com/miekg/dns => github.com/cilium/dns v1.1.51-0.20230303133941-d3bcb3008ed2
 	github.com/optiopay/kafka => github.com/cilium/kafka v0.0.0-20180809090225-01ce283b732b
-	go.universe.tf/metallb => github.com/cilium/metallb v0.1.1-0.20220829170633-5d7dfb1129f7
-	sigs.k8s.io/controller-tools => github.com/cilium/controller-tools v0.6.2
+	go.universe.tf/metallb => github.com/cilium/metallb v0.9.6
+	sigs.k8s.io/controller-tools => github.com/cilium/controller-tools v0.12.1
```

Comment:
It's rather unfortunate that renovate updates the replace directives. It should not as they will deviate from the ones that are specified by Cilium. I don't know if we can tell renovate to ignore replace directives, that's a problem we also face in the cilium/hubble repository.

Reply:
yes yes we can and I need to do it